databuilder/ehrql-tutorial-examples/project.yaml

```diff
@@ -0,0 +1,11 @@
+version: "3.0"
+
+expectations:
```

Comment:
We also have to have `expectations` in there, which isn't great. But the YAML doesn't work yet anyway.